src/DeviceInterfaceValues.tsx

```diff
@@ -571,6 +682,16 @@ export default (): React.ReactElement => {
               Publish Data
             </Button>
           )}
+        {astarte.token?.can(
+          'appEngine',
+          'POST',
```

Comment:
to fetch data, we need a `GET`, but considering the user opened this page, it already has the right permissions, no need to check this again